modules/deployments-ab-testing-lb.adoc

```diff
@@ -0,0 +1,258 @@
+  annotations:
+    haproxy.router.openshift.io/balance: roundrobin
+spec:
+  host: ab-example.my-project.my-domain
+  to:
+    kind: Service
+    name: ab-example-a
+    weight: 10
+  alternateBackends:
+  - kind: Service
+    name: ab-example-b
+    weight: 15
+...
+----
+
+[id='deployments-ab-testing-lb--{context}']
+== Managing weights using the web console
+
+.Procedure
+
+. Navigate to the Route details page (Applications/Routes).
+
+. Select *Edit* from the Actions menu.
+
+. Check *Split traffic across multiple services*.
+
+. The *Service Weights* slider sets the percentage of traffic sent to each service.
++
+For traffic split between more than two services, the relative weights are
+specified by integers between 0 and 256 for each service.
++
+Traffic weightings are shown on the *Overview* in the expanded rows of the
+applications between which traffic is split.
+
+[id='deployments-ab-testing-lb-cli-{context}']
+== Managing weights using the CLI
+
+.Procedure
+
+. To manage the services and corresponding weights load balanced by the route,
+use the `oc set route-backends` command:
++
+----
+$ oc set route-backends ROUTENAME \
+    [--zero|--equal] [--adjust] SERVICE=WEIGHT[%] [...] [options]
+----
++
+For example, the following sets `ab-example-a` as the primary service with
+`weight=198` and `ab-example-b` as the first alternate service with a
+`weight=2`:
++
+----
+$ oc set route-backends ab-example ab-example-a=198 ab-example-b=2
+----
++
+This means 99% of traffic is sent to service `ab-example-a` and 1% to
+service `ab-example-b`.
++
+This command does not scale the DeploymentConfigs. You might be required to do
+so to have enough Pods to handle the request load.
+
+. Run the command with no flags to verify the current configuration:
++
+----
+$ oc set route-backends ab-example
+NAME                    KIND     TO           WEIGHT
+routes/ab-example       Service  ab-example-a 198 (99%)
+routes/ab-example       Service  ab-example-b 2   (1%)
+----
+
+. To alter the weight of an individual service relative to itself or to the
+primary service, use the `--adjust` flag. Specifying a percentage adjusts the
+service relative to either the primary or the first alternate (if you specify
+the primary). If there are other backends, their weights are kept proportional
+to the changed.
++
+For example:
++
+----
+$ oc set route-backends ab-example --adjust ab-example-a=200 ab-example-b=10
+$ oc set route-backends ab-example --adjust ab-example-b=5%
+$ oc set route-backends ab-example --adjust ab-example-b=+15%
+----
++
+The `--equal` flag sets the `weight` of all services to `100`:
++
+----
+$ oc set route-backends ab-example --equal
+----
++
+The `--zero` flag sets the `weight` of all services to `0`. All requests then
+return with a 503 error.
++
+[NOTE]
+====
+Not all routers may support multiple or weighted backends.
+====
+
+
+[id='deployments-ab-one-service-multi-dc-{context}']
+== One service, multiple DeploymentConfigs
```

Comment:
@mfojtik I cannot get this procedure working. Original content came from https://docs.openshift.com/container-platform/3.11/dev_guide/deployments/advanced_deployment_strategies.html#routes-load-balancing-for-AB-testing and I tried fixing it but can't figure it and left it as is. Any thoughts?